ajax/libs/bootstrap-year-calendar/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "bootstrap-year-calendar",
+  "description": "A fully customizable year calendar widget, for Bootstrap!",
+  "homepage": "https://www.bootstrap-year-calendar.com/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Paul-DS/bootstrap-year-calendar"
+  },
+  "author": "Paul-DS <@Paul-DS> (https://taylorhakes.com/)",
+  "license": "Apache-2.0",
+  "filename": "js/bootstrap-year-calendar.min.js",
+  "keywords": [
+    "calendar",
+    "bootstrap"
+  ],
+  "npmName": "bootstrap-year-calendar",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "+(js|css)/**/bootstrap-year-calendar*"
+      ]
+    }
+  ]
+}
```

Comment:
@PeterDaveHello How do I use `fixFormat.js` for my local respository? Need some help with that.